src/index.tsx

```diff
@@ -358,6 +360,43 @@ function deleteSelectedElements() {
   }
 }
 
+function save() {
+  if (elements && elements.length > 0) {
+    const items = [...elements];
+    for (const item of items) {
+      item.isSelected = false;
+    }
+
+    localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify(items));
+  }
+
+  window.removeEventListener("beforeunload", onUnload);
+}
+
+function restore() {
+  const el = localStorage.getItem(LOCAL_STORAGE_KEY);
+
+  if (el) {
+    const items = JSON.parse(el);
```

Comment:
You probably want to add a try catch here, we want to fail silently if the data is corrupted